app/packages/plugins/src/index.ts

```diff
@@ -259,6 +282,7 @@ type PluginActivator = (props: any) => boolean;
 
 type PanelOptions = {
   allowDuplicates?: boolean;
+  pinned?: boolean;
```

Comment:
It is already in `panelOptions`, though.

It's a must in a sense that I want plugin authors to have a say in what is pinned by default and what is not. Open to being persuaded otherwise.

Reply:
Hmm... really? I believe it was intended to be the private option of spacenode. It should preferably be not controlled by the panel itself. I.e. opening a panel should not make it never closeable imo.

Reply:
Hmm, okay. I think there are valid use cases for wanting a panel to be pinned by default, but I'm okay with it not being exposed.